app/bundles/LeadBundle/Event/DoNotContactRemoveEvent.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\LeadBundle\Event;
+
+use Mautic\LeadBundle\Entity\Lead;
+use Symfony\Component\EventDispatcher\Event;
+
+class DoNotContactRemoveEvent extends Event
```

Comment:
I don't see a reason why this class should not be final. It doesn't have to be mocked as it does not use any services. What do you think?
```suggestion
final class DoNotContactRemoveEvent extends Event
```